app/src/main/java/org/oppia/android/app/profile/AdminAuthActivityPresenter.kt

```diff
@@ -99,6 +99,7 @@ class AdminAuthActivityPresenter @Inject constructor(
   private fun setTitleAndSubTitle(binding: AdminAuthActivityBinding?) {
     when (activity.intent.getIntExtra(ADMIN_AUTH_ENUM_EXTRA_KEY, 0)) {
       AdminAuthEnum.PROFILE_ADMIN_CONTROLS.value -> {
+        activity.title = context.getString(R.string.admin_auth_activity_access_controls_title)
         binding?.adminAuthToolbar?.title =
           context.resources.getString(R.string.administrator_controls)
```

Comment:
Why is this not also ``admin_auth_activity_access_controls_title`` for consistency?

Reply:
Answered below with the toolbar question.